### DISCUSSION TITLE
Set first day of the week in calendar panel

### DISCUSSION THREAD

**Discussion by uphillbattle** [home-assistant/frontend#5941] [Other feature requests] [answered]
The current calendar view has Sunday as the first day of the week. In Europe, Monday is considered the first day of the week. It would be great if the user could select which day is the first of the week.

**Comment by p4sI** [comment]
I invested some more time and it seems that you need to have import `@fullcalendar/core/locales-all`
as described in the example (https://fullcalendar.io/docs/locale)
```
import { Calendar } from '@fullcalendar/core';
import allLocales from '@fullcalendar/core/locales-all';
...
let calendar = new Calendar(calendarEl, {
  locales: allLocales,
  locale: 'fr' // the initial locale
});
...
calendar.setOption('locale', 'pt-br');
```
I only debugged it in my browser and instead of just setting `locale` to `"de"` I set it to the object from https://github.com/fullcalendar/fullcalendar/blob/master/packages/core/src/locales/de.ts and everything worked as expected, with a week starting on Monday.

Seems like https://github.com/home-assistant/frontend/blob/dev/src/panels/calendar/ha-full-calendar.ts#L162 is the file which need to be adapted.
@zsarnett maybe you can support here?

**Comment by Moonbase59** [comment]
This is really needed! And of course it would be just great if could be made dependant on the viewing user’s locale …

**Reply by iRonin** [reply]
💯

**Comment by mirekmal** [comment]
Please, please, do this explicitly configurable... we already struggle enough with time settings (12 vs 24 hours) being automatically detected and not working as users want.

**Comment by breti** [comment]
Please... this is still an issue in 0.115.

**Comment by sgtsilver** [comment]
Still an issue with 0.116.4

**Comment by zsarnett** [comment]
Nothing has been merged. Add a +1 but please do not comment :)

**Comment by zsarnett** [comment, marked as answer]
https://github.com/home-assistant/frontend/pull/7541

**Reply by uphillbattle** [reply]
Great! 🍾🎉

**Comment by TRusselo** [comment]
Since the update my calendar starts on Monday... GROSS.
in Canada the start of week on all Calendars is Sunday. 

Version | 0.118.0

**Reply by zsarnett** [reply]
What is your Locale set to in Your Profile? It should have never started in Monday though (before the release). At least I thought...

**Reply by TRusselo** [reply]
Profile....?  User profile... lets check there...  DAMNIT.    was set to language English (GB).

I was looking under server settings.... lat /long, timezone ect....
Thank you.
(smashes head on desk)

**Reply by uphillbattle** [reply]
Haha, for a short period of time, you conformed with ISO 8601 which says the week starts on Monday. ;)

**Reply by TRusselo** [reply]
work week starts on monday.
calendar week starts on sunday.

I know, us North Americans are weird.  But im Canadian, so, I'm not retarded.

**Comment by iRonin** [comment]
I'd like to be able to set the starting day of the week independently of the locale.

**Reply by ildar170975** [reply]
These are different fields, aren't they?
![image](https://github.com/home-assistant/frontend/assets/71872483/f96d5196-89e9-4826-a950-21f244215391)

**Reply by iRonin** [reply]
That's fantastic but where can I find these settings? 🧐

**Reply by uphillbattle** [reply]
Click here:
![image](https://github.com/home-assistant/frontend/assets/15340540/8a240309-a6c5-4048-b4ea-160d824fc8dc)

**Reply by iRonin** [reply]
Thanks! Not sure why but that's the first time I'm going to this settings 😂
